packages/react-components/src/alert/src/Alert.tsx

```diff
@@ -157,15 +164,15 @@ const variations = [
     { tone: "success", icon: <CheckIcon /> },
     { tone: "warning", icon: <WarningIcon /> },
     { tone: "critical", icon: <InfoIcon /> }
-];
+] as const;
```

Comment:
Souldn't it be a Record<string, Icon> ?

Reply:
It may be a personal opinion, but i prefer not creating types i really don't care about. In this case, either i do "as const" at the end of the array, or i create a temporary type like this

```
const variations: { tone: keyof typeof Role, icon: ReactElement }[] = [
    { tone: "info", icon: <NotificationIcon /> },
    { tone: "success", icon: <CheckIcon /> },
    { tone: "warning", icon: <WarningIcon /> },
    { tone: "critical", icon: <InfoIcon /> }
];
```
![image](https://user-images.githubusercontent.com/38871812/111627412-6784de00-87c5-11eb-91d6-b5647483abf3.png)

Also, the as const is a little bit more precise, since in theory we know it's not a random array of tone and icons, its an array where the first tone is info, the second is success ... 

You can see the typing generated with the as const 
![image](https://user-images.githubusercontent.com/38871812/111627609-9b600380-87c5-11eb-941b-ed242e2b0447.png)